cts/cli/regression.access_render.exp

```diff
@@ -1,7 +1,6 @@
 Created new pacemaker configuration
-Setting up shadow instance
-A new shadow instance was created.  To begin using it paste the following into your shell:
-  CIB_shadow=cts-cli ; export CIB_shadow
+A new shadow instance was created. To begin using it, enter the following into your shell:
+	export CIB_shadow=cts-cli
```

Comment:
The classic Bourne shell couldn't handle export and assignment in one step. Of course non-Bourne-compatible shells like csh don't even use export, so this isn't a big deal -- it can stay how you have it.

Reply:
I'm happy to change it back (can be separate PR). This is from a commit months ago though, it's just now being updated in the test output.

Reply:
no it's fine, anyone using an uncommon shell is used to dealing with such things